Makefile

```diff
@@ -1,13 +1,35 @@
-default: install
+.DEFAULT_GOAL = help
 
-generate:
-	go generate ./...
+GO-VERSION = 1.19.2
```

Comment:
Again, will this be updated automatically?

Reply:
Yes